sig-ml-developer-experience/README.md

```diff
@@ -1,30 +1,31 @@
 # ODH ML Developer Experience Special Interest Group
 
-SIG ML Developer Experience covers end-to-end experience for a data scientist, a data analyst or an application developer.
+ML Developer Experience SIG covers end-to-end user experience for the Open Data Hub interface.
 
 The [charter](./charter.md) defines the scope and governance of the ML Developer Experience Special Interest Group.
 
 ## Meetings
-[Monthly SIG meeting:](http://meet.google.com/ipx-ewwx-cup)  Every 4th Tuesday of the month.
-[Meeting notes and agenda](https://docs.google.com/document/d/1cGK5kkhp6SLhCc__pmLCfHEiCP-lZPcxpJTsz1nwaNY/edit).
+
+[Meeting notes and agenda](https://github.com/opendatahub-io/sig-ml-developer-experience/wiki).
 
 ## Leadership
 
 ### Chairs
+
 The Chairs of the SIG run operations and processes governing the SIG.
-- Jay Koehler
 - Kyle Walker
-- Erwan Granger
-- Romeo Kienzler
+- Andrew Ballantyne
+- 
 
```

Comment:
Removing Jay & Erwan from chairs as per their request.

Romeo has been inactive.

Reply:
Also added myself as I have been participating for some time. This was done on a call with Kyle.